src/store/playerReducer.js

```diff
@@ -1,4 +1,4 @@
-import {FETCH_CARDS_BEGIN, SET_PLAYED_CARD, SET_PLAY_WORD_AND_CARD, SET_PLAY_CARD, FETCH_CARDS_SUCCESS, FETCH_CARDS_FAILURE, REQUEST_PLAY_CARD, FINISH_PLAY_CARD, PLAY_WORD, MY_TURN, OTHERS_TURN} from './playerActions';
+import {RESET_PLAYER_STATE, FETCH_CARDS_BEGIN, SET_PLAYED_CARD, SET_PLAY_WORD_AND_CARD, SET_PLAY_CARD, FETCH_CARDS_SUCCESS, FETCH_CARDS_FAILURE, REQUEST_PLAY_CARD, FINISH_PLAY_CARD, PLAY_WORD, MY_TURN, OTHERS_TURN} from './playerActions';
```

Comment:
Do reduce the size of the imports, you could have these variables as part of some sort of `types` object.

Reply:
Just realised that should have said "to", not "do". That wasn't supposed to be a command!